dora/src/baseline/cannon/codegen.rs

```diff
@@ -49,6 +56,9 @@ pub struct CannonCodeGen<'a, 'ast: 'a> {
 
     pub cls_type_params: &'a TypeList,
     pub fct_type_params: &'a TypeList,
+
+    pub bytecode_to_address: HashMap<usize, usize>,
```

Comment:
I think it would be safer if the HashMap would be `HashMap<BytecodeIdx, usize>`.